dashboard/app/controllers/pd/csf_certificate_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Pd::CsfCertificateController < ApplicationController
+  before_action :authenticate_user!
+  load_resource :enrollment, class: 'Pd::Enrollment', find_by: :code, id_param: :enrollment_code
+
+  def generate_certificate
+    image = create_certificate_image2(
+      dashboard_dir('app', 'assets', 'images', 'pd_workshop_certificate_csf.png'),
+      @enrollment.try(:name) || '',
```

Comment:
`Pd::Enrollment.name` is [deprecated](https://github.com/code-dot-org/code-dot-org/blob/9ccf51f5d9679ca1104b54826b44eb6c04c2a044/dashboard/app/models/pd/enrollment.rb#L160). Use [full_name](https://github.com/code-dot-org/code-dot-org/blob/9ccf51f5d9679ca1104b54826b44eb6c04c2a044/dashboard/app/models/pd/enrollment.rb#L164-L168) instead.

As an aside, we need to remove name and other no longer used fields at some point.